go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/go-openapi/strfmt v0.19.3
 	github.com/go-openapi/swag v0.19.5
 	github.com/go-openapi/validate v0.19.4
-	github.com/stretchr/testify v1.7.0
-	gopkg.in/yaml.v2 v2.4.0
+	github.com/google/uuid v1.1.1
+	github.com/stretchr/testify v1.5.1
+	gopkg.in/yaml.v2 v2.2.8
```

Comment:
Please revert the changes, we upgraded testify and yaml using dependabot. And I'm not sure about why uuid was removed, but in case we do need it, please add it again.

Reply:
done, the dependencies are now up to date again